### DISCUSSION TITLE
Images not being displayed

### DISCUSSION THREAD

**Discussion by lmcole** [AntonyCorbett/OnlyM#475] [Q&A]
Has anyone had the problem where the play button for the images is not highlighted so that the image can be displayed?

**Comment by AntonyCorbett** [comment]
@lmcole A couple of things to check:
- Is a Monitor specified in the OnlyM Display settings?
- Is a video already playing (or other non-image media)?

**Reply by lmcole** [reply]
No monitor is specified and no video or other non-image media is displayed on the monitor besides the Yeartext.

**Reply by AntonyCorbett** [reply]
You must specify a monitor in OnlyM settings.

**Reply by lmcole** [reply]
Fantastic, thanks a million. 
I will tell the brothers the next time we are at meeting to do that

I appreciate your suggestion.
Will let you know what happens if that's ok

**Reply by AntonyCorbett** [reply]
Yes, sure. OnlyM just needs to know which monitor to use for its media output.

**Reply by AntonyCorbett** [reply]
See the wiki for more information.

https://github.com/AntonyCorbett/OnlyM/wiki

**Reply by lmcole** [reply]
Thanks :)

**Reply by lmcole** [reply]
Just to provide an update, we haven't had any images that were not in the JW Library app but may have some for the weekend meeting on Saturday.
I will try again to see what happens with getting them displayed properly.
I appreciate very much your assistance.

I am guessing it's safe to call you 'brother"?

**Reply by AntonyCorbett** [reply]
Yes.

**Comment by lmcole** [comment]
Good morning, Brother Corbett,

I wanted to let you know that as I fill in for the Circuit Overseer and was visiting a congregation last week the brothers didn't know how to display the . images I had with the service talk that is given after the abbreviated watchtower study.
So I introduced them to OnlyM.
They were stunned speechless because it solved so many of the issues they had displaying images.

My reason for this message is to thank you very much for making the program available.

Your brother,
Lavardo Colebrooke
***@***.***
(242) 525-2064

________________________________
